### ISSUE TITLE
Saving raw files with wildcards

### ISSUE THREAD

**Issue by materialvision** [introlab/odas#129]
Hi. Thanks for an amazing project. Is there any way to configure odaslive to save filename with a wildcard, for instance with date and time added to the separated.raw and postfiltered.raw filenames? 

It would be great as they are now overwritten when a new recording is started which is not ideal for my purpose.

**Comment by FrancoisGrondin** [comment]
Good point! I'm working on the new release; I'll keep this in mind. Thank you for your feedback!
